src/controllers/authController.js

```diff
@@ -0,0 +1,134 @@
+const bcrypt = require('bcrypt');
+const { User } = require('../models/auth');
+require('dotenv').config();
+const { createTokens } = require('../jwt.js'); 
+
+const login = async (req, res) => {
+  
+  const { email, password } = req.body;
+
+  try {
+
+    // checking if the user exist
+    const user = await User.findOne({ email });
+    if (!user) {
+      return res.status(401).json({ error: 'Invalid email or password' });
+    }
+
+    // checking if the password valid
+    const isPasswordValid = await bcrypt.compare(password, user.password);
+    if (!isPasswordValid) {
+      return res.status(401).json({ error: 'Invalid  email or password' });
+    }
+
+    // create JWT access token
+    const accessToken = createTokens(user);
+
+    // save the JWT token to the cookies
+    res.cookie('access-token', accessToken, {
```

Comment:
We need to change the name of the cookie to 'jwt` instead of `access-token`

Reply:
why?